### PR TITLE
[PHP] Antidot: fix dependency issues in composer json

### PR DESCRIPTION
**Pull request by kpicaza**
Hi @waghanza, I see that there is some issue running Antidot Framework inner the benchmark. 
image

I updated the composer.json file to get at least the minimum required versions.

Thanks for your great work and for maintaining this excellent project;-D

### CODE REVIEW

File: php/antidot/composer.json [hunk 1]
```diff
@@ -13,13 +13,12 @@
     "ext-json": "*",
     "antidot-fw/framework": "~0.2.1",
     "antidot-fw/antidot-react-psr15": "*",
-    "antidot-fw/cli": "*",
+    "antidot-fw/cli": "^1.1",
```
Comment (waghanza) [inline php/antidot/composer.json:16]: might the keyword `@stable` could work ?

**Review by waghanza — COMMENTED** [review]
I will update results, after merge

File: php/antidot/composer.json (hunk 1 above)
Comment (waghanza) [reply php/antidot/composer.json:16]: nevermind, could not

**Review by waghanza — COMMENTED** [review]
if you may invite me @kpicaza, I'll update results

**Comment by kpicaza** [comment]
Sorry @waghanza, I'm debugging in more depth because the server falls after some stress tests, If you don't mind I prefer to check better the issue before merging, and for the moment moving the PR to draft, sorry for the inconvenience.

image

**Comment by kpicaza** [comment]
I think the issue is related to keep-alive connections on React HTTP server as described here https://github.com/reactphp/http/pull/405#issuecomment-820388257. As a workaround, I will keep the react/http version fixed to v1.2.0. 

The performance is 2x better after v1.3.0, from 20k to 40k in Antidot Framework. But after some WRK hits the server crashes;-(.

**Review by waghanza — APPROVED** [review]

**Pull request merged**